pkgs/by-name/fm/fm-go/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, stdenv
+}:
+
+let
+  pname = "fm";
+  version = "0.16.0";
+  src = fetchFromGitHub {
+    owner = "mistakenelf";
+    repo = "fm";
+    rev = "v${version}";
+    hash = "sha256-wiACaszbkO9jBYmIfeQpcx984RY41Emyu911nkJxUFY=";
+  };
+  vendorHash = "sha256-AfRGoKiVZGVIbsDj5pV1zCkp2FpcfWKS0t+cTU51RRc=";
+in
+buildGoModule {
+  inherit pname version src vendorHash;
```

Comment:
```suggestion
buildGoModule rec {
  pname = "fm";
  version = "0.16.0";

  src = fetchFromGitHub {
    owner = "mistakenelf";
    repo = "fm";
    rev = "v${version}";
    hash = "sha256-wiACaszbkO9jBYmIfeQpcx984RY41Emyu911nkJxUFY=";
  };

  vendorHash = "sha256-AfRGoKiVZGVIbsDj5pV1zCkp2FpcfWKS0t+cTU51RRc=";
```

Reply:
I do not like `rec`.
So I reworked the expression.